### ISSUE TITLE
The img2img results are full of color noise

### ISSUE THREAD

**Issue by NEFUJoeyChen** [Stability-AI/stablediffusion#67]
![00000](https://user-images.githubusercontent.com/50317721/204469569-6624bf55-efb8-4b0c-a25e-9b037362515a.png)
The above figure is an example. After the step is set to 500, there is still no clear result.

**Comment by dogboydog** [comment]
Was getting the same but I think  the README example does not  show the correct config to use?
Here is an example that worked for me without noise. Download the 512-base-ema.ckpt [from here](https://huggingface.co/stabilityai/stable-diffusion-2-base/resolve/main/512-base-ema.ckpt) 

 `python scripts/img2img.py --prompt "A fantasy landscape, trending on artstation" --init-img ~/Pictures/example.png  --ckpt 512-base-ema.ckpt --config configs/stable-diffusion/v2-inference.yaml`

**Comment by b0bsl3d** [comment]
try changing you sampler from PLMS to DPM and see if you get sane results. I worked for me. you can do that at the command line by adding "--dpm" to your txt2img line

**Comment by tonydavis629** [comment]
I am having the same problem. There is no --dpm parameter in img2img.py.

**Comment by b0bsl3d** [comment]
yes, doesn't look like  img2img allows you to specify another sampler. if you look in the code (scripts\img2img.py) you should see:

     device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
    model = model.to(device)

    **sampler = DDIMSampler(model)**

I guess you could experiment with substituting another sampler invocation there - if you are getting the excessive noise from img2img, but I also wonder if its another problem. I have not seen oddities from img2img, only txt2img with PLMS sampler.

**Comment by andresberejnoi** [comment]
> try changing you sampler from PLMS to DPM and see if you get sane results. I worked for me. you can do that at the command line by adding "--dpm" to your txt2img line

Do you know why the PLMS sampler does not work in this case? I also found this same issue with the txt2img.py script. I get noise unless I use DPM or the default ones.

**Comment by fahmidme** [comment]
Try adding the config flag `--config configs/stable-diffusion/v2-inference.yaml`. It made the output much better for me.
